app/src/docs/v1.api-spec.yaml

```diff
@@ -588,6 +620,17 @@ components:
         permissionWebadeNrosDms:
           type: boolean
           example: false
+    AcronymUsersList:
+      type: array
+      items:
+        $ref: "#/components/schemas/AcronymUserMapping"
+    AcronymUserMapping:
+      type: object
+      properties:
```

Comment:
I think we should add an `"acronym": "MSSC"` property because when I look at the other two /acronyms/* endpoints they both have the input parameter also being emitted in the return JSON object - it's more mainly for consistency with the other existing endpoints, and it doesn't hurt to return back to the caller an extra confirmation that the requested object is what they actually wanted.